connector/oidc/oidc.go

```diff
@@ -186,7 +186,8 @@ func (c *oidcConnector) HandleCallback(s connector.Scopes, r *http.Request) (ide
 	}
 
 	var claims struct {
-		Username      string `json:"name"`
+		Name          string `json:"name"`
```

Comment:
This internal variable name has been changed to be more consistent. This means we need to change all the connectors in order to remain backwards compatible. If we don't update the connectors they will be setting the `username` claim instead of the `name` claim.